### DISCUSSION TITLE
Question: recommended way of using images and working in local?

### DISCUSSION THREAD

**Discussion by icougil** [mmistakes/minimal-mistakes#2593] [Q&A] [answered]
Hi!

I was having a look, and I saw that the recommended way of posting images is using a `{% raw %}` tag:
https://github.com/mmistakes/minimal-mistakes/blob/master/docs/_posts/2010-08-05-post-image-standard.md

Example: `{% raw %}<img src="{{ site.url }}{{ site.baseurl }}/assets/images/filename.jpg" alt="">{% endraw %}`

It is still the recommended way of doing it? Then what I have to do be able to see my images rendered properly in local? 

I'm not able to see them properly because they are rendered like this (without replacing any of the variables neither (`{{ site.url }}` nor `{{ site.baseurl }}`):

`<img src="{{ site.url }}{{ site.baseurl }}/assets/images/reluctant.jpeg" alt="alt">`

![image](https://user-images.githubusercontent.com/2370634/86586757-b452bc80-bf88-11ea-873f-75ed79a1b52e.png)

Thank you in advance!

**Comment by mmistakes** [comment]
If you're seeing `{{ site.url }}{{ site.baseurl }}` in dev tools, then something is off with your source files. Are you sure you're using those variables in documents that have YAML front matter? Jekyll won't render those liquid variabls unless you do that.

Either way, closing as this isn't a theme specific issue. Direct Jekyll related questions/concerns to their support forum.

https://talk.jekyllrb.com/

**Comment by icougil** [comment, marked as answer]
Oh I found the issue!
Instead of having a look on the reference guide that you have published in your page [here](https://mmistakes.github.io/minimal-mistakes/post%20formats/post-image-standard/), I was checking the [source page](https://github.com/mmistakes/minimal-mistakes/blob/master/docs/_posts/2010-08-05-post-image-standard.md), and therefore I was reading the instructions in Markdown not properly rendered instead of in HTML 
That's why the `{% raw %}` makes no sense 😅
